programs/uxd/src/lib.rs

```diff
@@ -10,7 +10,6 @@ pub mod events;
 pub mod instructions;
 pub mod mercurial_utils;
 pub mod state;
-pub mod test;
```

Comment:
the test code was being linked with the main program 😓

Reply:
Huh 😅